houston/queries.go

```diff
@@ -339,6 +339,9 @@ mutation UpdateDeployment($deploymentId: Uuid!, $payload: JSON!, $executor: Exec
       username
       roleBindings {
         role
+		deployment {
+			id
+		}
```

Comment:
```suggestion
		id
	}
```

Reply:
I think I need to look at tabs vs spaces in my machine 🤔 , thanks for catching it

Reply:
may be add validation to this file somehow?

Reply:
because it is a multi-line string, and we have spaces & tabs inside the string makes it difficult to format/validate. Maybe there might be a better way to store graphql query strings, will have a look at it